mm2src/mm2_test_helpers/src/for_tests.rs

```diff
@@ -951,7 +960,6 @@ pub fn jst_sepolia_conf() -> Json {
             "type": "ERC20",
             "protocol_data": {
                 "platform": "ETH",
-                "chain_id": ETH_SEPOLIA_CHAIN_ID,
```

Comment:
looking at it. this actually used to look better (having chain_id in the protocol specific discriminator instead of the higher level)

Reply:
This was wrongly placed and not used actually, `chain_id` is kept at the higher level for backward compatibility for v1 methods, but ERC20 tokens take the chain_id from platform coin in v2 activations. So in time `chain_id` shouldn't be in ERC20 tokens configs at all, which makes the most sense.

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/commit/2b44429a3522596d12128316b85270b9a067ce2a